src/ebmc/show_formula_solver.h

```diff
@@ -0,0 +1,70 @@
+/*******************************************************************\
+
+Module: Show Formula Solver
+
+Author: Daniel Kroening, kroening@kroening.com
+
+\*******************************************************************/
+
+/// \file
+/// Show Formula Solver
+
+#ifndef EBMC_SHOW_FORMULA_SOLVER_H
+#define EBMC_SHOW_FORMULA_SOLVER_H
+
+#include <solvers/stack_decision_procedure.h>
+
+class show_formula_solvert : public stack_decision_proceduret
+{
+public:
+  // console
+  show_formula_solvert();
+
+  // file
+  explicit show_formula_solvert(std::ostream &_out) : out(_out), console(false)
+  {
+  }
+
+  void set_to(const exprt &, bool value) override;
+  exprt handle(const exprt &) override;
+  exprt get(const exprt &) const override;
+  void print_assignment(std::ostream &) const override
+  {
+  }
+
+  std::string decision_procedure_text() const override
+  {
+    return "show-formula solver";
+  }
+
+  std::size_t get_number_of_solver_calls() const override
+  {
+    return 0;
+  }
+
+  ~show_formula_solvert()
+  {
+  }
```

Comment:
Not necessary?

Reply:
To avoid a warning -- even though the object is never deleted via the base type

Reply:
But then it could be `~show_formula_solvert() = default;`?

Reply:
Sure